packages/p2p/src/components/my-profile/my-profile.jsx

```diff
@@ -35,177 +52,383 @@ const MyProfile = observer(() => {
-                                <div className='my-profile__stats-cell-header'>
-                                    {localize('Buy ({{currency}})', { currency: general_store.client.currency })}
+                    <React.Fragment>
+                        <div className='my-profile__content'>
+                            <div className='my-profile__stats'>
+                                <div className='my-profile__stats-cell-separator' />
+                                <div className='my-profile__stats-cell'>
+                                    <div className='my-profile__stats-cell-header'>{localize('Total Trades')}</div>
+                                    <div className='my-profile__stats-cell-info'>{total_orders_count || '-'}</div>
                                 </div>
-                                <div className='my-profile__stats-cell-info'>{buy_orders_count || '-'}</div>
-                            </Table.Cell>
-                            <div className='my-profile__stats-cell-separator' />
-                            <Table.Cell className='my-profile__stats-cell'>
-                                <div className='my-profile__stats-cell-header'>
-                                    {localize('Sell ({{currency}})', { currency: general_store.client.currency })}
+                                <div className='my-profile__stats-cell-separator' />
+                                <div className='my-profile__stats-cell'>
+                                    <div className='my-profile__stats-cell-header'>
+                                        {localize('Buy / Sell ({{currency}})', {
+                                            currency: general_store.client.currency,
+                                        })}
+                                    </div>
+                                    <div className='my-profile__stats-cell-info'>
+                                        {buy_orders_count || '-'}/{sell_orders_count || '-'}
+                                    </div>
                                 </div>
-                                <div className='my-profile__stats-cell-info'>{sell_orders_count || '-'}</div>
-                            </Table.Cell>
-                            <div className='my-profile__stats-cell-separator' />
-                            <Table.Cell className='my-profile__stats-cell'>
-                                <div className='my-profile__stats-cell-header'>
-                                    {localize('Buy / Sell limit ({{currency}})', {
-                                        currency: general_store.client.currency,
-                                    })}
+                                <div className='my-profile__stats-cell-separator' />
+                                <div className='my-profile__stats-cell'>
+                                    <div className='my-profile__stats-cell-header'>
+                                        {localize('Buy / Sell limit ({{currency}})', {
```

Comment:
For this, you can use a ternary operator, does not need `MobileWrapper` and `DesktopWrapper`.